aws-iot-mitigationaction/inputs/inputs_1_create.json

```diff
@@ -0,0 +1,17 @@
+{
+    "ActionName": "CfnContractTest",
+    "RoleArn": "arn:aws:iam::072938559174:role/Admin",
+    "ActionParams":
+        {
```

Comment:
Let's run `pre-commit run --all-files`, that should reformat all the json files to be consistent.

Reply:
Done